soroban-sdk/test_wasms/README.md

```diff
@@ -0,0 +1,6 @@
+# test_wasms
+
+Files contained in this directory are used in a few SDK tests that are sensitive
+to Wasm content changes.
+
+`test_contract_data.wasm` is a build of `contract_data`  test contract.
```

Comment:
We don't need to prebuild the wasm and commit it. All of CI runs `make build` before the tests, and we have other tests that read wasms straight out of the target directory, so should follow suit.

Reply:
We do need to store this Wasm for the sake of the test stability. CI already manages to build Wasm that slightly differs in size on different platforms, and in the future any SDK change would lead to this test breaking. The intention of this test not to cover how SDK builds Wasm, but rather that SDK uses the env functionality properly, so I think storing a prebuilt Wasm will save us a lot of maintenance pain (again, not mention that it would require some hacks to make it pass today). The alternative is to not cover the exact numbers at all, but I don't really like that - the test is a nice demonstration of what things are metered.

Reply:
Right, but we already have other tests that make this assumption, so we're making this test work differently to the other tests in this repo. Let's address that problem in the future when it becomes a problem and address it holistically?\

Reply:
Or are you saying we already have that issue right now with this test?

Reply:
Yes, we do have divergence right now. I'm not sure if fixing this is the right use of time - even if Wasm was consistent between all the build configurations, it's still annoying that unrelated changes might require updating this test (I'd prefer it only to need updates when the protocol and thus metering changes - this is how the env tests behave and this feature is unfortunately pretty tightly coupled with the env internals).

Reply:
👍🏻